docs/getting-started/interact-with-the-cli.md

```diff
@@ -14,6 +14,32 @@ Upon success, all definitions in the **definitions.py** (or whatever you choose
 
 After applying new resource definitions, you can use the **GET** command to see the status of the resources you applied. A resource with the status **READY** is available for serving.
 
+## DASH Command
+
+```
+featureform dash
+```
+
+The **DASH** command is used to access the featureform dashboard. It returns a URL to the locally hosted dashboard. 
+
+Example of the dashboard’s URL:
+
+Running on http://localhost:3000/
```

Comment:
Change this to: "The dashboard can be viewed at `http://localhost:3000` in your browser